actions/invite_actions.test.ts

```diff
@@ -0,0 +1,538 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
In the new and old version of the test file, the only real change is that the return format for the actions is now wrapped in an object with a `data` key. This accompanying change in result format was done in the implementation to make the actions compatible with `ActionResult`, which in turn made converting all the places that use this action to Typescript easier.